src/guide/contributing-to-an-open-jdk-project.md

```diff
@@ -52,7 +58,9 @@ Even though we strive to unify how things are done within OpenJDK, different are
 * [CSR Process](https://wiki.openjdk.org/display/csr/Main)
 :::
 
-Java and the JDK are very popular products, and just about every Java developer out there has an idea or two for how to enhance something. And (obviously not referring to you) believe it or not, not every idea is a good idea. Even though many ideas are indeed good, we must be quite restrictive on what we actually include into the JDK. There are many reasons for this.
+Only the best patches submitted will actually make it all the way into the JDK code base. The goal is not to take in the maximum number of contributions possible, but rather to accept only the highest-quality contributions. The JDK is used daily by millions of people and thousands of businesses, often in mission-critical applications, and so we can't afford to accept anything less than the very best.
```

Comment:
"Only the best" is not really a defining criteria here. This paragraph really overlaps with the following paragraph. First the idea must be appropriate and agreed that a patch is needed, then the quality of the patch itself will be assessed using various criteria depending on the exact nature of the patch.

Reply:
I rephrased it somewhat.